tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -791,26 +765,27 @@ def test_rwo_pvc_fencing_node_prolonged_network_failure(
                 f"App pod with name {pod_obj.name} did not reach Running state"
             )
 
-        # Wait for mon and osd pods to reach Running state
-        selectors_to_check = [constants.MON_APP_LABEL, constants.OSD_APP_LABEL]
-        for selector in selectors_to_check:
-            assert ceph_cluster.POD.wait_for_resource(
-                condition=constants.STATUS_RUNNING, selector=selector,
-                resource_count=3, timeout=1800, sleep=60
-            ), (
-                f"3 expected pods with selector {selector} are not in Running state"
-            )
+        if not external_mode:
+            # Wait for mon and osd pods to reach Running state
+            selectors_to_check = [constants.MON_APP_LABEL, constants.OSD_APP_LABEL]
+            for selector in selectors_to_check:
+                assert ceph_cluster.POD.wait_for_resource(
+                    condition=constants.STATUS_RUNNING, selector=selector,
+                    resource_count=3, timeout=1800, sleep=60
```

Comment:
what if the cluster was expended? mon and OSD count will not be 3.

Reply:
ack for OSD, we can check for the actual count.
for Mon, we would still want to check for 3, since we don't support more than 3 mons. And even in case of previous OCS version where we had 5 mons, the failed mons might not be scheduled on nodes where other mons are already running.